starters/next-wordpress-starter/lib/Pages.js

```diff
@@ -25,7 +25,7 @@ export async function getAllPagesUri() {
 export async function getLatestPages() {
 	const query = gql`
 		query LatestPagesQuery {
-			pages(first: 10) {
+			pages(first: 12) {
```

Comment:
@mitchellmarkoff I should have caught this in review. first: 12 in the query here is limiting the results to 12 even in cases where we have substantially more content. Instead we want to get all pages, but only show 12 results per page.

Quickly tried removing first: 12, but then was only seeing 10 results.